app/schemas.py

```diff
@@ -306,6 +317,7 @@ class Meta:
         )
         load_only = ['_bearer_token', 'bearer_token']
         strict = True
+        load_instance = True
```

Comment:
Is this still necessary with load_instance and include_relationships set on BaseSchema?  Ditto subsequent occurrences.